### DISCUSSION TITLE
테스트용 Heroku 앱  배포했습니다.

### DISCUSSION THREAD

**Discussion by artemismars** [artemismars/mern_blog#52] [Announcements]
![image](https://user-images.githubusercontent.com/75290655/153708685-8e9b4e4c-f84b-424a-ad74-57735f868f1c.png)

main 브랜치로 머지되는 코드들은 헤로쿠 앱에 자동 반영됩니다.

헤로쿠 무료 서비스를 이용하여 배포된 앱은 접속 시간이 550시간으로 제한되어 있습니다.
그래서 아래와 같은 사용 절차를 거쳤으면 합니다.

1. 자신이 구현한 코드를 로컬에서 정상 작동하는 지 테스트 후 PR 오픈.
2. 정기 미팅에서 전체 동의 후에 `fe_dev` -> `main` merge.
3. `Heroku` 앱에 접속하여 테스트 시행.
4. 테스트 완료 및 실패 여부 확인 후 접속 해제.

상황을 봐서 위 절차가 효율적이지 않을 경우 대안책을 모색해보는 것으로 하겠습니다.

Heroku 테스트용 앱 주소
https://mern-blog-heroku.herokuapp.com/

**Comment by artemismars** [comment]
@Dhyunlee @TaewookKwak @5witchkr
확인 부탁드립니다.

**Reply by donghyunami** [reply]
확인했습니다

**Reply by TaewookKwak** [reply]
네 이해는 잘 안가지만, 저 서버 사용 시간 제한이 있다는 말씀이죠?

**Reply by 5witchkr** [reply]
확인했습니다 제출양식이 어떻게되나영?

**Reply by artemismars** [reply]
> 네 이해는 잘 안가지만, 저 서버 사용 시간 제한이 있다는 말씀이죠?

@TaewookKwak 맞습니다. 무료 버전 이용중이라 제한있어요!

서비스 상태가 2가지 있습니다.
1. 활성화: 앱 접속시 무료 이용시간 소모
2. Sleep (비활성화): 비접속시 무료이용시간 소모 정지

**Reply by artemismars** [reply]
> 확인했습니다 제출양식이 어떻게되나영?

@5witchkr 어떤 제출 양식 말씀하시는건가요?

**Comment by artemismars** [comment]
@Dhyunlee @TaewookKwak @5witchkr
본문 2번 더 구체적으로 설명했습니다.
![image](https://user-images.githubusercontent.com/75290655/153736885-03efa86b-4b3b-4651-af7a-a2b64df68979.png)
